scripts/benchmark_all.sh

```diff
@@ -95,6 +87,14 @@ do
   # Checkout and build from scratch, tracking the compile time.
   git checkout "$commit"
   git submodule update --init --recursive
+
+  if [ -f ../resources/bolt.fdata ]
```

Comment:
I guess it’s for development?
No need to change it now, but that should be changed later …

Reply:
No, I think that this should stay there, so we have to correct build configuration for the optimization step. If bolt is not wanted, one can just remove the bolt.fdata file.

Reply:
Let’s discuss it later ..